src/pubkey.h

```diff
@@ -205,4 +210,13 @@ struct CExtPubKey {
     bool Derive(CExtPubKey& out, unsigned int nChild) const;
 };
 
+/** Users of this module must hold an ECCVerifyHandle. The constructor and
+ *  destructor of these are not allowed to run in parallel, though. */
```

Comment:
Yes, I guess passing secp256k1_context_verify explicitly explicitly would be too disruptive. Could we document somewhere that it would be ideal but we just don't want to do it right now? Maybe an example doing it "the right way" (a static getter or something) would be useful too.